### ISSUE TITLE
More documentation

### ISSUE THREAD

**Issue by pcsswamin** [Esri/Compare-Maps-Template#15]
Kelly,

For developers to follow your lead and customize this template further, it would really help to add more developer focused documentation to this template and others that you create. Right now- all info is more towards config than customize.

A quick walk-through through the logic/architecture of this template. Even thoughts such as  - your take on using gulp would be beneficial. Any such developer oriented info would really help developers use this template as a jump-off point to a custom one.

Thanks
Subu

**Comment by kellyhutchins** [comment]
Good idea. I'll work on updating this doc to add more developer centric info.
